libnmstate/netinfo.py

```diff
@@ -28,9 +28,13 @@ def show():
 
 def interfaces():
     info = []
-    for dev in nm.device.list_devices():
-        devinfo = nm.device.get_device_common_info(dev)
+
+    devices_info = [(dev, nm.device.get_device_common_info(dev))
+                    for dev in nm.device.list_devices()]
+
+    for dev, devinfo in devices_info:
```

Comment:
I needed to pass on the list of devices (devices_info) in order to detect the bridge slaves and the port slaves.
Hopefully, libnm can provide a "get_slaves" functionality like it has for the bond, then we will be able to avoid this.